persist-cli-tests/src/test/resources/test-src/input/tool_test_db_push_25/generated/rainier/generated_client.bal

```diff
@@ -0,0 +1,468 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+
+// This file is an auto-generated file by Ballerina persistence layer for rainier.
+// It should not be modified by hand.
+
+import ballerina/persist;
+import ballerina/sql;
+import ballerina/time;
+import ballerinax/mysql;
+
+const 'BUILDING = "building";
+const 'DEPARTMENT = "department";
+const 'EMPLOYEE = "employee";
+const 'ORDER_ITEM = "orderitem";
+const 'WORKSPACE = "workspace";
+
+public client class RainierClient {
+    *persist:AbstractPersistClient;
+
+    private final mysql:Client dbClient;
+
+    private final map<persist:SQLClient> persistClients;
+
+    private final record {|persist:Metadata...;|} metadata = {
+        "building": {
+            entityName: "Building",
+            tableName: `Building`,
+            fieldMetadata: {
+                'buildingCode: {columnName: "buildingCode", 'type: string},
+                'city: {columnName: "city", 'type: string},
+                'state: {columnName: "state", 'type: string},
+                'country: {columnName: "country", 'type: string},
+                'postalCode: {columnName: "postalCode", 'type: string},
+                'type: {columnName: "type", 'type: string}
+            },
+            keyFields: ["buildingCode"]
+        },
+        "department": {
+            entityName: "Department",
+            tableName: `Department`,
+            fieldMetadata: {
+                deptNo: {columnName: "deptNo", 'type: string},
+                deptName: {columnName: "deptName", 'type: string}
+            },
+            keyFields: ["deptNo"]
+        },
+        "employee": {
+            entityName: "Employee",
+            tableName: `Employee`,
+            fieldMetadata: {
+                empNo: {columnName: "empNo", 'type: string},
+                firstName: {columnName: "firstName", 'type: string},
+                lastName: {columnName: "lastName", 'type: string},
+                birthDate: {columnName: "birthDate", 'type: time:Date},
+                gender: {columnName: "gender", 'type: string},
+                hireDate: {columnName: "hireDate", 'type: time:Date},
+                'departmentDeptNo: {columnName: "departmentDeptNo", 'type: string},
+                'workspaceWorkspaceId: {columnName: "workspaceWorkspaceId", 'type: string}
```

Comment:
Do we need generated foreign keys to escape?

Reply:
updated in [ab8528a](https://github.com/ballerina-platform/persist-tools/pull/137/commits/ab8528a6d28a13e761569ec8c5a4239d4fb57963)